### ISSUE TITLE
wazuh-csyslogd - 100% CPU if format=json

### ISSUE THREAD

**Issue by StrilGit** [wazuh/wazuh#17604]
Using the Centos 7 OVA with wazuh 4.4.4 with only one agent.

```
<ossec_config>
  <syslog_output>
    <server>x.x.x.x</server>
    <port>yyyy</port>
    <format>json</format>
   </syslog_output>
</ossec_config>
```

If format is set to json, the process wazuh-csyslogd is using 100% of one CPU core.
If format is changed to default or cef, cpu drops to <1%

Did you ever see that issue?
Best wishes

**Comment by leba-atr** [comment]
I am experiencing the very same issue as well. It is definitely related to the JSON format. Running `wazuh-csyslogd` with `-dd` yields this log:
```
root@wazuh:/var/ossec/bin# ./wazuh-csyslogd -f -dd
2023/08/11 15:32:42 wazuh-csyslogd[1422] debug_op.c:70 at _log(): DEBUG: Logging module auto-initialized
2023/08/11 15:32:42 wazuh-csyslogd[1422] main.c:107 at main(): DEBUG: Wazuh home directory: /var/ossec
2023/08/11 15:32:42 wazuh-csyslogd[1422] pthreads_op.c:45 at CreateThreadJoinable(): DEBUG: Thread stack size set to: 8192 KiB
2023/08/11 15:32:42 wazuh-csyslogd[1422] main.c:168 at main(): DEBUG: Chrooted to directory: /var/ossec, using user: wazuh
2023/08/11 15:32:42 wazuh-csyslogd[1422] main.c:179 at main(): INFO: Started (pid: 1422).
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:73 at OS_CSyslogD(): DEBUG: JSON file queue connected.
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:85 at OS_CSyslogD(): DEBUG: Resolving server hostname: 172.18.0.1
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:93 at OS_CSyslogD(): INFO: Forwarding alerts via syslog to: '172.18.0.1:514'.
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
2023/08/11 15:32:42 wazuh-csyslogd[1422] csyslogd.c:110 at OS_CSyslogD(): DEBUG: jqueue_next()
[...]
```
This log line is emitted by the main loop of csyslogd (specifically lines 110 and 111): 
https://github.com/wazuh/wazuh/blob/5fa2996c20178a7b70d3866e031273ecd6dfca7c/src/os_csyslogd/csyslogd.c#L99..L136

**Comment by vikman90** [comment]
@StrilGit @leba-atr thank you for reporting this issue. We also were notified by a support customer and fixed that in version 4.5.2.

Related PR: https://github.com/wazuh/wazuh/pull/18472.

Best regards.
